components/expenses/EditExpenseDialog.tsx

```diff
@@ -428,7 +531,7 @@ const EditExpenseTitle = ({ expense, onSubmit }) => {
   });
 
   return (
-    <FormikZod schema={schema} initialValues={expense} onSubmit={values => onSubmit(schema.parse(values))}>
+    <FormikZod schema={schema as any} initialValues={expense} onSubmit={values => onSubmit(schema.parse(values))}>
```

Comment:
It seems that removing this cast isn't introducing any error, I suppose it was fixed since?

```suggestion
    <FormikZod schema={schema} initialValues={expense} onSubmit={values => onSubmit(schema.parse(values))}>
```

Reply:
Ah, yes, thanks!